elide-core/src/main/java/com/yahoo/elide/core/utils/TypeHelper.java

```diff
@@ -148,4 +156,25 @@ public static String getFieldAlias(String tableAlias, String fieldName) {
                         .map(ClassType::of)
                         .collect(Collectors.toSet());
     }
+
+    /**
+     * Parses input string and return arguments as map.
+     *
+     * @param argsString String to parse for arguments.
+     * @return A Map of argument's name and value.
+     * @throws UnsupportedEncodingException
+     */
+    public static Map<String, String> parseArguments(String argsString) throws UnsupportedEncodingException {
+        Map<String, String> arguments = new HashMap<>();
+
+        if (!isEmpty(argsString)) {
+
+            Matcher matcher = FILTER_ARGUMENTS_PATTERN.matcher(argsString);
```

Comment:
We should move the FILTER_ARGUMENTS_PATTERN to this file.  Does this function make sense in TypeHelper?